DataProcessing/datax-host/src/main/scala/datax/input/BlobPointerInput.scala

```diff
@@ -39,9 +39,9 @@ object BlobPointerInput {
     (new StructType).add("BlobPath", StringType)
   }
 
-  private lazy val saRegex = s"""wasbs?://[\\w-]+@([\\w\\d]+)${BlobProperties.BlobHostPath}/.*""".r  
+  private lazy val saRegex = s"""(?:wasbs?|abfss?)://[\\w-]+@([\\w\\d]+)(${BlobProperties.BlobHostPath}|${BlobProperties.DfsHostPath})/.*""".r
```

Comment:
Nice find! Adding non capture specifier here ?: will help since we dont want to include it in the list of results.